serverlessworkflow/modules/ROOT/pages/eventing/consume-produce-events-with-knative-eventing.adoc

```diff
@@ -154,8 +154,54 @@ The following table lists the configuration property related to Knative sink gen
 |`default`
 |This property indicates the name of the default Knative broker that is deployed in the Kubernetes namespace. This broker is used as the reference to create the Knative triggers, which are responsible to delegate the events that the workflow service consumes.
 
+| `mp.messaging.incoming.kogito_incoming_stream.path`
+| `/` (root path)
+|This property indicates the HTTP path where the workflow application will listen for the CloudEvents in the default incoming channel.
+
+| `mp.messaging.incoming.<event type name>.path`
+| `/` (root path)
+| This property indicates the HTTP path where the workflow application will listen for the CloudEvents in the specific given channel name. The channel name is the event `type` as defined in the Serverless Workflow events definition.
+
```

Comment:
why not merging both and add a specific mention for `kogito_incoming_stream` in the description ?

Reply:
+1 to that

Reply:
Honestly, I think it's more clear this way. You add both possibilities, and they are indeed different. You can control the specific channel path and the shared one if you have both.